### ISSUE TITLE
Table TblCons, data deleted

### ISSUE THREAD

**Issue by victorsh1968** [mRemoteNG/mRemoteNG#1711]
all the data just disappeared from the table..., i restored the database and again in few seconds again all the data disappeared .....

HELP !!!!

I'm using MSSQL for the repository database.

**Comment by farosch** [comment]
Please provide a meaningful issue by filling out the template.
These information are crucial for us to pinpoint to your issue and solve it.
I will reopen this issue as soon as you provide us with the necessary information.

**Comment by victorsh1968** [comment]
it's not important anymore for me as i left the company and working in a new place, so i don't care anymore.

I opened this in 12/3, wow you guys are fast, today is 31/5...

We put a trigger in the table when identifying complete data delete there will be a rollback, but from time to time this also didn't helped as there were few row-by-row delete batches so the trigger didn't stopped this. 

We also created a snapshot on every one rounded hour just to be able to return fast the data, we had a lot of data deletion and we just inserted back the lost data.

Also, we double-saved us by configure the table data changes only by one member in the team, others connected with a read-only account from the application to the repository database to get the connections.

Thank you, that's it

Vic(S)
